Automattic/wp-calypso
# Stats: Update Videos summary page

#### Proposed Changes

* Add a feature flag and modernise the `Video` summary details page
* Extend the Horizontal bar component and its card component to:
   * render additional value columns by accepting a new parameter
   * add an option for an alternate header displaying column names below the card's title
   * update the width of the value column to accommodate larger numbers (a fixed width is needed for the label ellipsis)
   * add an option for a flexible name of the main item column (value column name is already flexible)
   * hide displaying data is loading property is passed (unlike other components, Videos summary page already have partial data when navigating to a page but still loads more)

I plan to update click events for each number in a separate PR.

#### Testing Instructions



* Navigate to the main Stats page on the live branch
* append `?flags=stats/new-video-summary` to the URL
* visit `Videos` summary page by clicking `View all` on the component on the `Traffic` page
* make sure that the component loads properly
* Bonus: smoke test other horizontal bar component on different screen sizes

| Before | After |
| --- | --- |
| ![SCR-20230201-hkp](https://user-images.githubusercontent.com/112354940/215917975-8038600d-29e4-4118-8181-58d7618d67f2.png) | ![SCR-20230201-hgq](https://user-images.githubusercontent.com/112354940/215918006-375f16b0-52a0-40b5-9aba-2a97b1454a79.png) |


#### Pre-merge Checklist





Related to #71018
